### PR TITLE
feat: add ncurses

### PR DESCRIPTION
**Pull request by wep21**
add [ncurses](https://invisible-island.net/ncurses)

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (ncurses) have been updated in this PR. Please review the changes.

File: modules/ncurses/6.4.20221231/patches/add_build_file.patch [hunk 1]
```diff
@@ -0,0 +1,430 @@
+diff --git BUILD.bazel BUILD.bazel
+new file mode 100644
+index 00000000..fcaad983
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,424 @@
++# Copyright 2020 Google LLC
```
Comment (fmeum) [inline modules/ncurses/6.4.20221231/patches/add_build_file.patch:7]: Could you link upstream for this file? It's probably the most complex one we would be adding to the registry. Do you think that it's sufficiently generic?

Comment (wep21) [reply modules/ncurses/6.4.20221231/patches/add_build_file.patch:7]: This comes from [here](https://github.com/hdl/bazel_rules_hdl/blob/main/dependency_support/net_invisible_island_ncurses/bundled.BUILD.bazel).

**Comment by wep21** [comment]
Needs skip url stability check

**Review by meteorcloudy — APPROVED** [review]

File: modules/ncurses/6.4.20221231/patches/add_build_file.patch (hunk 1 above)
Comment (Wyverald) [reply modules/ncurses/6.4.20221231/patches/add_build_file.patch:7]: ping @fmeum . are we good to merge?

**Pull request merged**